src/env.js

```diff
@@ -73,17 +74,20 @@ export class Env {
      * Retrieves an environment variable without checking for its presence.
      * Optionally returns a default value if the environment variable doesn't
      * exist.
+     * @template [S=undefined]
      * @param {string} key The environment variable name to retrieve.
-     * @param {string} [defaultValue] The default value to return if the
+     * @param {S} [defaultValue] The default value to return if the
```

Comment:
I’m not super familiar with TypeScript types — can you explain what this change does?

Reply:
see https://github.com/humanwhocodes/env/pull/52#discussion_r937167801